app/assets/javascripts/discourse/app/components/user-stream-item.js

```diff
@@ -1,3 +1,8 @@
+/*
+ *  !!! DEPRECATED COMPNENT: DO NOT USE !!!
```

Comment:
```suggestion
 *  !!! DEPRECATED COMPONENT: DO NOT USE !!!
```

Should we use the standard deprecation mechanism here (that logs in console etc)?